README.md

```diff
@@ -2,7 +2,7 @@
 
 For full documentation, see <http://lookit.readthedocs.io> (source: <https://github.com/lookit/lookit-docs/>) 
 
-This is the codebase for the Lookit project, an online developmental laboratory. It includes an Experimenter platform for designing and administering research studies, and a participant-facing platform where users can signup and take part in studies. It is built using Django, PostgreSQL, and Ember.js (see Ember portion of codebase, [ ember-lookit-frameplayer](https://github.com/lookit/ember-lookit-frameplayer)), and is developed by [MIT](https://mit.edu) and [Center for Open Science](https://cos.io/).
+This is the codebase for the Lookit project, an online developmental laboratory. It includes a Researcher platform for designing and administering research studies, and a participant-facing platform where users can signup and take part in studies. It is built using Django, PostgreSQL, and Ember.js (see Ember portion of codebase, [ ember-lookit-frameplayer](https://github.com/lookit/ember-lookit-frameplayer)), and is developed by [MIT](https://mit.edu) and [Center for Open Science](https://cos.io/).
```

Comment:
this one can stay since it's referencing the name of the application rather than the users